cmd/download.go

```diff
@@ -47,7 +47,9 @@ func download(ctx *cmdutil.Ctx) error {
 
 			localAsset, _ := shopify.ReadAsset(ctx.Env, requestAsset.Key)
 			if localAsset.Checksum == requestAsset.Checksum && requestAsset.Checksum != "" {
-				ctx.Log.Printf("[%s] Skipped %s (%s)", colors.Green(ctx.Env.Name), colors.Blue(requestAsset.Key), localAsset.Checksum)
+				if ctx.Flags.Verbose {
```

Comment:
So it looks like this PR includes changes to download as well, you should include that in the PR description.

Reply:
I also committed 96344d7 for a similar purpose of hiding the skips when not in verbose mode.